fusion-cli/test/e2e/split/test.js

```diff
@@ -16,10 +17,21 @@ test('`fusion dev` CHUNK_ID instrumentation', async () => {
   const url = app.url();
   const resA = await request(`${url}/test-a`);
   const resB = await request(`${url}/test-b`);
-  const res = await request(`${url}/test`);
-  t.deepEqual(JSON.parse(res), [0]);
-  t.deepEqual(JSON.parse(resA), [0, 1]);
-  t.deepEqual(JSON.parse(resB), [0, 2]);
+  const resCombined = await request(`${url}/test-combined`);
+  const resTransitive = await request(`${url}/test-transitive`);
+  t.deepEqual(JSON.parse(resA), [0, 2]);
+  t.deepEqual(JSON.parse(resB), [0, 3]);
+  t.deepEqual(JSON.parse(resCombined), [0]);
+  t.deepEqual(JSON.parse(resTransitive), [1]);
```

Comment:
I added another top-level dependency (`"test-d"`), so these chunk ids got bumped:
chunk 0 contains `"test-a"`, `"test-b"`, and `"test-c"`
chunk 1 contains `"test-d"`
chunk 2 contains `"test-a"`
chunk 3 contains `"test-b"`